README.md

```diff
@@ -300,6 +300,13 @@ amadeus.shopping.hotelOffersByHotel.get({
 // Confirm the availability of a specific offer id
 amadeus.shopping.hotelOffer('XXX').get()
 
+
+// Retrieve flight order with ID
```

Comment:
I would add `The flight ID comes from the Flight Create Orders (in test environment it's temporary)`

Reply:
Fixed